src/EnergyPlus/DataSizing.cc

```diff
@@ -965,6 +945,31 @@ namespace DataSizing {
         }
     }
 
+    Real64 ZoneAirDistributionData::calculateEz(EnergyPlusData &state, int const ZoneNum) // Zone index
```

Comment:
New function to retrieve current air distribution effectiveness for calculating target dynamic ventilation rate.

Reply:
Looks good to me.